README.md

```diff
@@ -7,6 +7,7 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 ## Table of Contents
 
 - [Resources](#resources)
+  - [FreeCodeCamp](#FreeCodeCamp) 
```

Comment:
When Helon said 'Resources', theres already a title section for that. You shouldnt create a new one, just move the FreeCodeCamp item under the already existing 'Resources' title.

Reply:
And dont forget to add the actual link to the page between the parentesis, not the '#FreeCodeCamp' as you wrote, because that is not going to bring us to the web page when we click the link.